proto/ibc/core/channel/v1/tx.proto

```diff
@@ -454,5 +454,10 @@ message MsgPruneAcknowledgements {
   string signer     = 4;
 }
 
-// MsgPruneAcknowledgementsResponse defines the response type for the PruneAcknowledgements rpc.
-message MsgPruneAcknowledgementsResponse {}
\ No newline at end of file
+// MsgPruneStalePacketStateResponse defines the response type for the PruneStalePacketState rpc.
+message MsgPruneStalePacketStateResponse {
+  // Number of sequences pruned (might include both packet acknowledgements and packet receipts).
+  uint64 pruned_sequences = 1;
+  // Number of sequences left after pruning.
+  uint64 remaining_sequences = 2;
```

Comment:
I think we should add `total_` prefix to these fields. 

`pruned_sequences` to me suggests a `repeated` field whereas with the "total" prefix its explicit